modules/highgui/src/window_framebuffer.hpp

```diff
@@ -0,0 +1,134 @@
+#ifndef OPENCV_HIGHGUI_WINDOWS_FRAMEBUFFER_HPP
```

Comment:
Add license header: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure

Reply:
license added (the problem with the URL remains)